packages/interactive-code-block/project.json

```diff
@@ -14,7 +14,7 @@
 				"cwd": "dist/packages",
 				"commands": [
 					"cp ../../packages/interactive-code-block/README.md ./interactive-code-block/",
-					"rm interactive-code-block/assets/php_5*",
+					"rm -f interactive-code-block/assets/php_5*",
```

Comment:
Why only add `-f` in this line of code but not the other ones? And why update the `interactive-code-block`?

Reply:
I was getting an error during the build process due to the attempted removal of a non-existent file. `-f` prevents `rm` from throwing errors when you try to remove a file that isn't there. I've reverted it locally and the issue doesn't seem to be happening anymore.

Reply:
@adamziel See, that worries me. If someone still has that directory, and then moves to a version that doesn't have an `rm` call for it, then it won't be removed. I think we should **definitely** open the PR now, **BUT** I think we should hold off on actually merging it for a predetermined amount of time, so that this can run off. Let me know your thoughts.

Reply:
How would someone still have that directory after `git pull` and `npm install`?

Reply:
The way I understand git is that checking out a newer revision where a file is removed also removes that file locally. Am I getting that wrong?